packages/augur-test/src/tests/state/getter/users/get-profit-loss-and-summary.test.ts

```diff
@@ -820,6 +820,47 @@ describe('State API :: Users :: PL :: ', () => {
       },
     });
   });
+
+  test('Scenario 7: partial closing initial position', async () => {
```

Comment:
The "Scenario #" convention is from a specific google doc. I'd put this test in its own `describe` block.

Reply:
ok